week6/Q1.py

```diff
@@ -0,0 +1,120 @@
+class Move:
+	'''
+	Class which stores pairs of positions from and to which a car is moved.
+
+	Fields:
+		_from: int, position from which element is moved
+		_to: int, position to which element is moved
+	'''
+	def __init__(self, from_value, to_value):
+		self._from = from_value
+		self._to = to_value
+
+
+def _find_car_on_empty_place(init_list, final_list, empty_place, moves):
```

Comment:
It's a little bit strange that function that calls "find" also *moves* smth. May be it would be better if this function will return desired index of this car, and then from somewhere else you will call "move" function with this data?
Or call this function `find_and_move`.